org.eclipse.xtext.web/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "xtext",
 	"private": true,
 	"devDependencies": {
-		"mocha": "6.1.4",
+		"mocha": "^7.2.0",
```

Comment:
Why is the `^` symbol needed?

Reply:
ask npm

Reply:
I do not know much about npm, but I rely on you.

Reply:
Are the changes in the `package-lock.json` file necessary/on purpose?